Improve Docker setup guide - Closes #1275 (#1300)
LiskHQ/lisk-docs: modules/ROOT/pages/setup/docker.adoc

== Configuring Lisk Core

Update the RPC configuration to enable the WebSocket API of Lisk Core.

> To do this, create a custom configuration file, for example by copying one of the default config files:

Reviewer: ```suggestion
To accomplish this, create a custom configuration file, for example by copying one of the default config files:
```

./home/lisk/lisk-core/
[source,bash]
----
cp config/mainnet/config.json custom-config.json
----

Add the following option to `custom-config.json`:

.custom-config.json
[source,js]
----
{
    // ...
    "rpc": {
        "enable": true,
        "mode": "ws",
        "port": 8080,
        "host": "0.0.0.0" // <1>
    },
    // ...
}
----

> <1> Allows remote servers to connect to the Lisk Core WS API.
> If this option is not set, it defaults to `127.0.0.1`.

Reviewer 2: Kindly consider adding this information as a `Note:`. Please feel free to adjust the document syntaxes.
```suggestion
Note: Allows remote servers to connect to the Lisk Core WS API. The `_host_` property defaults to `127.0.0.1`.
```

Author: this is a code comment, indicated by the prepended `<1>`, which is also referenced in the code snippet. So I wouldn't make it a note here.

Use the Lisk Core CLI to start Lisk Core with the desired configurations.

> The `--network` flag determines the blockchain network Lisk Core connects to.

Author: I think this changes the meaning. It's more like the node connecting to the network, not the network connecting to the node.

> The `--network` flag determines the blockchain network Lisk Core connects to.

Reviewer: OK, I can see how it could be interpreted the wrong way here, Ok then let's leave it as it is Mona, although generally, it is not good to end a sentence with 'to'

> The `--network` flag defines which blockchain network Lisk Core will connect to.

Reviewer 2: ```suggestion
The `--network` flag determines the blockchain network Lisk Core connects to.
```

`--api-ws` enables the WebSocket API of the node, if not already enabled in the config.json.

[source,bash]
----
lisk-core start --network mainnet --api-ws --config=/home/lisk/lisk-core/custom-config.json --overwrite-config
----

To connect to a different network, replace `mainnet` with either `testnet` or `devnet`.

== Setting up Lisk Service

[IMPORTANT]
====

> It is strongly recommended that you synchronize your Lisk Core node with the network **before** starting the Lisk Service.

Reviewer: ```suggestion
It is strongly recommended to synchronize your Lisk Core node with the network **before** starting the Lisk Service.
```

> ====

Reviewer 2: The table alignment doesn't seem to be not centred. Please check: https://github.com/LiskHQ/lisk-docs/blob/06ea2857bd76f6fc3d6ad413c5b081e49a858926/modules/ROOT/pages/setup/docker.adoc#3-setting-up-lisk-service

Author: that is only an issue on GitHub, on the docs-UI it will be rendered centered :)
